packages/desktop/views/dashboard/governance/views/ProposalDetailsView.svelte

```diff
@@ -202,7 +199,7 @@
     }
 
     onMount(async () => {
-        pollParticipationEventStatus($selectedProposal?.id).then(() => (statusLoaded = true))
+        pollProposalData($selectedProposal?.id).then(() => (statusLoaded = true))
```

Comment:
because we do this fetch only here in the details view, the pill in the proposalslist only updates after we went to the details once.
Maybe we should have 1 `await getParticipationEventStatusForEvent(eventId)` for each event one time on load.